pkg/subctl/cmd/verify.go

```diff
@@ -62,14 +62,20 @@ var verifyCmd = &cobra.Command{
 	Args: func(cmd *cobra.Command, args []string) error {
 		return checkValidateArguments(args)
 	},
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
 		configureTestingFramework(args)
 		if verifyConnectivity || verifyAll {
-			verifyconnectivity.RunE2E(&testing.T{})
+			if !verifyconnectivity.RunE2E(&testing.T{}) {
+				return fmt.Errorf("Connectivity E2E failed")
```

Comment:
use exitOnError, you can probably refactor exit on error into two functions:
```
// exitOnError will print your error nicely and exit in case of error
func exitOnError(message string, err error) {
	if err != nil {
		exitWithErrorMsg(fmt.Sprintf("%s: %s", message, err)
	}
}


func exitWithErrorMsg(message string) {
		fmt.Fprintln(os.Stderr, message)
		fmt.Fprintln(os.Stderr, "")
		PrintSubctlVersion(os.Stderr)
		fmt.Fprintln(os.Stderr, "")
		os.Exit(1)
}
```

Reply:
Done